src/pages/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { PageSection, PageSectionVariants } from '@patternfly/react-core';
+import { WorkspaceDetails } from '../../components/WorkspaceDetails';
+
+const Workspaces: React.FC = () => {
+  const title = 'Workspace Details';
+
+  return (
+    <>
+      <Helmet>
```

Comment:
Move this to `WorkspaceDetails` so we can include the workspace name (which will be helpful for bookmarking, etc)?

Reply:
So I'm torn over what should be the best pattern moving forward.  Looks like files in the `pages` directory determine the overall wrapper/layout for the page.  

Wondering if instead of putting the code including `helmet` in the component, we pull the workspace name from useParams.

Reply:
I don't have a strong opinion on either approach (`<Helmet>` is designed to allow lower level components to override, but that doesn't mean it's necessarily a good pattern), but do think we should include the workspace name in `<title>`.